File: python/Template/complete/HaikuTwo.py
from Template import Template

class HaikuTwo(Template):
    '''
    implements the methods from superclass Template to write a haiku
    My life,
    How much more of it remains
    The night is brief
    '''

    def write_first_line(self):
        return "My life,"

    def write_second_line(self):
        return "How much more of it remains"

    def write_third_line(self):
        return "The night is brief"

File: java/Template/complete/HaikuTwo.java
public class HaikuTwo extends Template{
  /*
  Overrides the abstract methods from superclass Template to write a haiku
  My life,
  How much more of it remains
  The night is brief
  */

  @Override
  public String write_first_line(){
    return "My life,";
  }

  @Override
  public String write_second_line(){
    return "How much more of it remains";
  }

  @Override
  public String write_third_line(){
    return "The night is brief";
  }
}
